js/sign.js

```diff
@@ -0,0 +1,38 @@
+export function noneEmailCheck(value) {
+  return value.length >= 1;
+}
+
+export function failEmailCheck(value) {
+  return /^[A-Za-z0-9_\.\-]+@[A-Za-z0-9\-]+\.[A-za-z0-9\-]+/.test(value);
+}
+
+export function nonePasswordCheck(value) {
+  return value.length >= 1;
+}
+
+export function failPasswordCheck(value) {
+  return /^(?=.*?[A-Z])(?=.*?[a-z])(?=.*?[0-9])(?=.*?[#?!@$%^&*-]).{8,}$/.test(
+    value
+  );
+}
+
+export function noneNameCheck(value) {
+  return value.length >= 1;
+}
+
+export function samePasswordCheck(password1, password2) {
+  if (password1 === password2) {
+    return true;
+  }
+  return false;
+}
```

Comment:
boolean 표현식은 이렇게 쓰는 편이 일반적입니다.

```suggestion
export function samePasswordCheck(password1, password2) {
  return password1 === password2
}
```